hazelcast/vector.py

```diff
@@ -33,6 +51,11 @@ def __init__(self, value: Any, vectors: Union[Vector, List[Vector]]) -> None:
 
     @property
     def vector(self) -> Optional[Vector]:
+        """Returns the vector associated with this Document
```

Comment:
I wonder how to access all vectors in the List if user requires that

Reply:
It's `document.vectors`.